apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -293,6 +309,16 @@ export const WorkoutEditor = ({
             Use without saving
           </Button>
 
+          {canBeDeleted && (
+            <Button
+              onClick={() => setShowDeleted(true)}
+              leftIcon={<Icon as={Trash} />}
+              color={'firebrick'}
```

Comment:
Should probably use `colorScheme` instead 🤓 
```suggestion
              colorScheme="red"
```
https://chakra-ui.com/docs/components/button/usage